src/vercel.js

```diff
@@ -47,25 +47,25 @@ const init = () => {
 			]
 
 			metadata.forEach((item) => {
-				command += ` -m "${ item }"`
+				commandArguments.push(`--meta="${ item }"`)
```

Comment:
Shorthand with `=` doesn't work, `-m=bla` but `--meta=bla` does. We can't use a space, `--meta bla` since spawn would be confused by that.